lib/blockscore/base.rb

```diff
@@ -60,8 +60,16 @@ def persisted?
       !id.nil? && !deleted?
     end
 
+    def deleted?
```

Comment:
@tjchambers I enabled mutant to run against each commit and the majority of the things it highlights are missing specs for return values.

Keep in mind the change I made only flags missing mutations in *code changed in the last commit*. It doesn't do anything about existing code -- I suspect we'll need a different plan of attack to resolve missing mutations from existing code. This is more about drawing a line in the sand and not allowing mutation coverage to drop at all in new code.